### PR TITLE
Init

### PR DESCRIPTION
**Pull request by Alexander8933**

### CODE REVIEW

**Comment by Alexander8933** [comment]
test

File: src/Shop.java [hunk 1]
```diff
@@ -0,0 +1,167 @@
+import java.io.*;
+import java.util.ArrayList;
+
+ class Shop {
+    private ArrayList<Bike> bikes = new ArrayList<Bike>();
+
+     public static void main(String[] args) {
+        new Shop().launchShop();
```
Comment (sIvanovKonstantyn) [inline src/Shop.java:8]: move this main method to the `LaunchShop` class

File: src/Shop.java [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    }
+
+    private void launchShop() {
+        getBikesFromTxt();
+        showMenu();
+    }
+    //test2
```
Comment (sIvanovKonstantyn) [inline src/Shop.java:15]: remove comment

File: src/Shop.java [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+                bufferedWriter.write(bike.getValueBikeInSaveTxt());
+                bufferedWriter.newLine();
+            }
+        } catch (IOException ex) {
+            System.out.println("File BikeBase.txt May be damaged");
+            ex.printStackTrace();
+        }
+    }
+
+    private void addBikeFromTxt(String bikeLine) {
```
Comment (sIvanovKonstantyn) [inline src/Shop.java:159]: `addBike` is enough. Remove the rest if the name

Comment (sIvanovKonstantyn) [reply src/Shop.java:159]: bikeLine -> bikeString

File: src/Shop.java [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+            if (choice.equals("n")) {
+                return;
+            }
+        }
+        if (bikeSize == 0) {
+            System.out.println("Empty base");
+        }
+    }
+
+    private void getBikesFromTxt() {
```
Comment (sIvanovKonstantyn) [inline src/Shop.java:130]: Move this method to a separate class "BikesDao" and rename as "findAll"

File: src/Shop.java [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+        } catch (FileNotFoundException e) {
+            System.out.println("File BikeBase.txt Not Found ");
+            e.printStackTrace();
+        } catch (IOException ex) {
+            System.out.println("File BikeBase.txt May be damaged");
+            ex.printStackTrace();
+        }
+    }
+
+    private void saveBikesInTxt() {
```
Comment (sIvanovKonstantyn) [inline src/Shop.java:146]: Move this method to a separate class "BikesDao" and rename as "saveAll(List<Bike> bikes)"

File: src/Shop.java [hunk 6]
```diff
@@ -0,0 +1,167 @@
+import java.io.*;
+import java.util.ArrayList;
+
+ class Shop {
+    private ArrayList<Bike> bikes = new ArrayList<Bike>();
```
Comment (sIvanovKonstantyn) [inline src/Shop.java:5]: Move this list to a `BikesDao` class

Comment (sIvanovKonstantyn) [reply src/Shop.java:5]: Add next methods to a `BikesDao` class:
`Bike findOne(int id);`
`void update(int id, Bike  bike);`
`int save(Bike  bike);`
`void delete(int id)`

**Review by sIvanovKonstantyn — CHANGES_REQUESTED** [review]

File: BikeBase.txt [hunk 7]
```diff
@@ -0,0 +1,4 @@
+comanche tomahawk#1300#6#
```
Comment (sIvanovKonstantyn) [inline BikeBase.txt:1]: Write in commit messages, meaningful information regarding what you have done in this commit.
3 "Init" commits - not a good idea.

**Review by sIvanovKonstantyn — CHANGES_REQUESTED** [review]

File: BikeBase.txt (hunk 7 above)
Comment (sIvanovKonstantyn) [inline BikeBase.txt:1]: Where are the packages?

File: src/Shop.java (hunk 1 above)
Comment (Alexander8933) [reply src/Shop.java:8]: ок

File: src/Shop.java (hunk 2 above)
Comment (Alexander8933) [reply src/Shop.java:15]: ок

File: src/Shop.java (hunk 3 above)
Comment (Alexander8933) [reply src/Shop.java:159]: ok

File: BikeBase.txt (hunk 7 above)
Comment (Alexander8933) [reply BikeBase.txt:1]: ok.

File: src/Shop.java (hunk 4 above)
Comment (Alexander8933) [reply src/Shop.java]: ok

File: src/Shop.java (hunk 6 above)
Comment (Alexander8933) [reply src/Shop.java]: ok

File: src/Shop.java (hunk 5 above)
Comment (Alexander8933) [reply src/Shop.java]: ok

File: BikeBase.txt (hunk 7 above)
Comment (Alexander8933) [reply BikeBase.txt:1]: ok

Comment (sIvanovKonstantyn) [reply BikeBase.txt:1]: [Update Shop.] != a well named commit

Comment (sIvanovKonstantyn) [reply BikeBase.txt:1]: Not ok. Read about packages and naming conventions regarding them. Find examples. Describe to me in the chat (telegram) what you learned before making any changes.

File: src/Dao/BikesDao.java [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+
+import java.io.*;
+import java.util.ArrayList;
+
+class BikesDao {
+
+    private ArrayList<Bike> bikes = new ArrayList<Bike>();
+    private File file = new File("BikeBase.txt");
+
+    void findAll() {
```
Comment (sIvanovKonstantyn) [inline src/Dao/BikesDao.java:13]: it's not a `find all` because you not really find them, but mostly load.
So rename this method to "loadData"

Comment (Alexander8933) [reply src/Dao/BikesDao.java]: ok

**Review by sIvanovKonstantyn — APPROVED** [review]

**Pull request merged**